Module-1 | Day-3/JavaScript-DOM.js

```diff
@@ -0,0 +1,13 @@
+var colorSelected;
+function colorValue(color){
+	colorSelected=color;
+}
+
+function colorMe(number){
+	document.getElementById(number).style.backgroundColor=colorSelected;
+}
+
+function refresh(){
+	window.location.reload();
```

Comment:
reload the screen??! you are supposed to reset the block colors, not refresh the whole page.